app/src/main/kotlin/com/simplemobiletools/calculator/helpers/CurrencyConverter.kt

```diff
@@ -0,0 +1,5 @@
+package com.simplemobiletools.calculator.helpers
```

Comment:
Why do you have two interfaces that implement the same method signature?

Reply:
I could remove CurrencyConverter and TaxCalculator interfaces here as the are not used anymore. They were combined into the MoneyCalculator interface in an effort to reduce the number of instance variables in MoneyActivity.

Reply:
Yea that would be good! Please remove this class.